.travis.yml

```diff
@@ -10,13 +10,14 @@ osx: &osx
    language: generic
 matrix:
    include:
-
-      - <<: *linux
-        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49
+#      - <<: *linux
```

Comment:
Why did you remove gcc-4.9?

Reply:
Because it crashes when setting up xorg-dev as a system dependency. It's the only image that does this.